builder-support/specs/dnsdist.spec

```diff
@@ -62,9 +62,6 @@ dnsdist is a high-performance DNS loadbalancer that is scriptable in Lua.
 %prep
 %autosetup -p1 -n %{name}-%{getenv:BUILDER_VERSION}
 
-# run as dnsdist user
-sed -i '/^ExecStart/ s/dnsdist/dnsdist -u dnsdist -g dnsdist/' dnsdist.service.in
```

Comment:
commit message doesn't mention this change, while it seems important enough to get into the changelog

Reply:
Indeed, and it should even have been a separate commit. I'll make it happen, thanks!

Reply:
I removed this part of the commit, and moved it to a separate PR: https://github.com/PowerDNS/pdns/pull/13088